app/assets/stylesheets/nav.scss

```diff
@@ -10,34 +16,63 @@ $button-color: #1E90FF;
   list-style-type: none;
 }
 
+.top-bar {
+  background-color: #3f3f3f;
```

Comment:
Color literals like `#3f3f3f` should only be used in variable declarations; they should be referred to via variable everywhere else.